File: net/NetTupleEngine/IntToken.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace it.mintlab.desktopnet.tupleengine
{
    class IntToken : IToken
    {
        private int valueInt;

        public IntToken(int number)
        {
            this.valueInt = number;
        }

        public int getValue()
        {
            return valueInt;
        }

        public override string ToString()
        {
            return valueInt.ToString();
        }

    }
}

File: trunk/java/JavaTupleEngine/src/it/mintlab/desktopjava/tupleengine/IntToken.java
package it.mintlab.desktopjava.tupleengine;

	public class IntToken implements IToken
	{
		private int valueInt;

		public IntToken(int number)
		{
			this.valueInt = number;
		}

		public final int getValue()
		{
			return valueInt;
		}

		@Override
		public String toString()
		{
			return (new Integer(valueInt)).toString();
		}

	}